multihost_test/Bugzillas/test_Regression.py

```diff
@@ -538,3 +538,249 @@ def test_bz1114081_1285547(self, multihost, create_backup):
+        :expectedresults:
+          1. Should succeed
+          2. Should succeed
+          3. Should succeed
+          4. Should succeed
+          5. Should succeed
+          6. Should succeed
+          7. Should succeed
+          8. Should succeed
+        """
+        client = multihost.client[0]
+        subuid = "/etc/subuid"
+        subgid = "/etc/subgid"
+        client.run_command(f"> {subuid}")
+        client.run_command(f"> {subgid}")
+        client.run_command("COLUMNS=1000 man login.defs | col -b > /tmp/anuj")
+        real_data = client.run_command("cat /tmp/anuj").stdout_text
+        for data1 in [subuid,
+                      "SUB_UID_MIN",
+                      "SUB_UID_MAX",
+                      "SUB_UID_COUNT",
+                      "SUB_GID_MIN",
+                      "SUB_GID_MAX",
+                      "SUB_GID_COUNT"]:
+            assert data1 in real_data
+        sub_uid_min, sub_uid_max, sub_uid_count = client.run_command("sed -n 's/.*SUB_GID_MIN, SUB_GID_MAX, "
+                                                                     "SUB_GID_COUNT[^0-9]*\([0-9]*\), \([0-9]*\) "
+                                                                     "and \([0-9]*\).*/\\1 \\2 \\3/p' "
+                                                                     "/tmp/anuj").stdout_text.split()
+        sub_gid_min, sub_gid_max, sub_gid_count = client.run_command("sed -n 's/.*SUB_GID_MIN, SUB_GID_MAX, "
+                                                                     "SUB_GID_COUNT[^0-9]*\([0-9]*\), \([0-9]*\) "
+                                                                     "and \([0-9]*\).*/\\1 \\2 \\3/p' "
+                                                                     "/tmp/anuj").stdout_text.split()
+        client.run_command("rm -vf /tmp/anuj")
+        subuid = "/etc/subuid"
+        subgid = "/etc/subgid"
+        user = "usertest1"
+        client.run_command(f"useradd {user}")
+        client.run_command(f"cat {subuid}")
+        client.run_command(f"cat {subgid}")
+
+        assert client.run_command(f"cut -d: -f1 {subuid}").stdout_text.split()[0] == user
+        assert client.run_command(f"cut -d: -f2 {subuid}").stdout_text.split()[0] == sub_uid_min
+        assert client.run_command(f"cut -d: -f3 {subuid}").stdout_text.split()[0] == sub_uid_count
+
+        assert client.run_command(f"cut -d: -f1 {subgid}").stdout_text.split()[0] == user
+        assert client.run_command(f"cut -d: -f2 {subgid}").stdout_text.split()[0] == sub_gid_min
+        assert client.run_command(f"cut -d: -f3 {subgid}").stdout_text.split()[0] == sub_gid_count
+
+        client.run_command(f"userdel -rf {user}")
+        with pytest.raises(Exception):
+            client.run_command(f"grep {user} {subuid}")
+        with pytest.raises(Exception):
+            client.run_command(f"grep {user} {subgid}")
+
+    @pytest.mark.tier1
+    def test_bz1671280_2(self, multihost, create_backup):
```

Comment:
Is there any actual difference between this test case and the previous one?

Reply:
sub_uid_min, sub_uid_max these values have diffrence

Reply:
This test cant be parametrized both have different steps .

Reply:
Ok, I've put both test cases side by side and they have more differences than just

> sub_uid_min, sub_uid_max these values have diffrence

Taking that into account I think it's fine to keep them as they are.